src/openrct2-ui/ride/Construction.cpp

```diff
@@ -24,6 +24,169 @@ using namespace OpenRCT2::TrackMetaData;
+        TrackElemType::LeftFlyerLargeHalfLoopInvertedUp,
+        TrackElemType::LeftFlyerLargeHalfLoopUninvertedDown,
+        TrackElemType::RightFlyerLargeHalfLoopUninvertedUp,
+        TrackElemType::RightFlyerLargeHalfLoopInvertedDown,
+        TrackElemType::RightFlyerLargeHalfLoopInvertedUp,
+        TrackElemType::RightFlyerLargeHalfLoopUninvertedDown,
+
+        // Twists
+        TrackElemType::LeftTwistDownToUp,
+        TrackElemType::LeftTwistUpToDown,
+        TrackElemType::RightTwistDownToUp,
+        TrackElemType::RightTwistUpToDown,
+        TrackElemType::LeftFlyerTwistUp,
+        TrackElemType::LeftFlyerTwistDown,
+        TrackElemType::RightFlyerTwistUp,
+        TrackElemType::RightFlyerTwistDown,
+
+        // Corkscrews
+        TrackElemType::LeftCorkscrewUp,
+        TrackElemType::LeftCorkscrewDown,
+        TrackElemType::RightCorkscrewUp,
+        TrackElemType::RightCorkscrewDown,
+        TrackElemType::LeftFlyerCorkscrewUp,
+        TrackElemType::LeftFlyerCorkscrewDown,
+        TrackElemType::RightFlyerCorkscrewUp,
+        TrackElemType::RightFlyerCorkscrewDown,
+        TrackElemType::LeftLargeCorkscrewUp,
+        TrackElemType::LeftLargeCorkscrewDown,
+        TrackElemType::RightLargeCorkscrewUp,
+        TrackElemType::RightLargeCorkscrewDown,
+
+        // Helixes
+        TrackElemType::LeftHalfBankedHelixUpSmall,
+        TrackElemType::RightHalfBankedHelixUpSmall,
+        TrackElemType::LeftHalfBankedHelixDownSmall,
+        TrackElemType::RightHalfBankedHelixDownSmall,
+        TrackElemType::LeftHalfBankedHelixUpLarge,
+        TrackElemType::RightHalfBankedHelixUpLarge,
+        TrackElemType::LeftHalfBankedHelixDownLarge,
+        TrackElemType::RightHalfBankedHelixDownLarge,
+        TrackElemType::LeftQuarterBankedHelixLargeUp,
+        TrackElemType::RightQuarterBankedHelixLargeUp,
+        TrackElemType::LeftQuarterBankedHelixLargeDown,
+        TrackElemType::RightQuarterBankedHelixLargeDown,
+        TrackElemType::LeftQuarterHelixLargeUp,
+        TrackElemType::RightQuarterHelixLargeUp,
+        TrackElemType::LeftQuarterHelixLargeDown,
+        TrackElemType::RightQuarterHelixLargeDown,
+
+        // Zero-G Rolls
+        TrackElemType::LeftZeroGRollUp,
+        TrackElemType::LeftZeroGRollDown,
+        TrackElemType::RightZeroGRollUp,
+        TrackElemType::RightZeroGRollDown,
+        TrackElemType::LeftLargeZeroGRollUp,
+        TrackElemType::LeftLargeZeroGRollDown,
+        TrackElemType::RightLargeZeroGRollUp,
+        TrackElemType::RightLargeZeroGRollDown,
+
+        // Barrel Rolls
+        TrackElemType::LeftBarrelRollUpToDown,
+        TrackElemType::LeftBarrelRollDownToUp,
+        TrackElemType::RightBarrelRollUpToDown,
+        TrackElemType::RightBarrelRollDownToUp,
+
+        // Tower
+        TrackElemType::TowerBase,
+        TrackElemType::TowerSection,
+
+        // Boosters
+        TrackElemType::Booster,
```

Comment:
My suggestion for ordering things is:
- functional pieces at the top
- miscellaneous nonfunctional pieces
- inversion pieces.

I think a good order might be:

- station
- brake
- booster
- photo
- rotation control toggle
- curved lifthill
- heartline transfers
- brake for drop
- s-bends
- helixes
- wooden water splash
- river rapids pieces
- spinning tunnel
- wooden reverser pieces
- reverser freefall & thrust air pieces
- loops
- twists
- barrel rolls
- zero g rolls
- corkscrews
- tower
- minigolf